src/main/java/programminglife/gui/controller/GuiController.java

```diff
@@ -179,6 +179,7 @@ private void initRecent() {
         }
         if (recentFile != null) {
             try (Scanner sc = new Scanner(recentFile)) {
+                menuRecent.getItems().clear();
```

Comment:
Thanks for adding this! I hadn't had any problems with duplicated, and I couldn't come up with a way to fix it. Well done!